File: src/test/java/nz/co/searchwellington/urls/UrlParserTest.java
package nz.co.searchwellington.urls;

import static org.junit.Assert.assertEquals;

import org.junit.jupiter.api.Test;

public class UrlParserTest {

	@Test
	public void canExtractHostFromFullyQualifiedUrl() {
		UrlParser parser = new UrlParser();
		
		assertEquals("wellington.gen.nz", parser.extractHostnameFrom("http://wellington.gen.nz/test/blah?q=123"));		
	}
	
}

File: src/test/scala/nz/co/searchwellington/urls/UrlParserTest.scala
package nz.co.searchwellington.urls

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class UrlParserTest {

  @Test
  def shouldExtractHostFromFullyQualifiedUrl(): Unit = {
    val parser = new UrlParser
    assertEquals(Some("wellington.gen.nz"), parser.extractHostnameFrom("http://wellington.gen.nz/test/blah?q=123"))
  }

}